src/client/components/CommentCount.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import _ from 'lodash';
+import { Text } from '@chakra-ui/react';
+
+interface Props {
+  commentCount: string;
+}
+
+export const CommentCount = (props: Props) => {
+  if (_.toInteger(props.commentCount) < 1) {
+    return <Text color={'gray.500'}>Comment</Text>;
+  } else if (_.toInteger(props.commentCount) == 1) {
```

Comment:
== isn't the same as ===. You want to use === for equality checks

Reply:
Also am i missing something? You're returning the same thing in every case? What's the point of the if conditions?